### PR TITLE
Create a weekly test job

### PR DESCRIPTION
**Pull request by gaborh-da**

### CODE REVIEW

**Comment by chrisnorris-da** [comment]
Why not nightly?

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,7 +38,13 @@ commands:
       - run:
           command: |
             ./scripts/install-daml.sh << parameters.version >>
-
+  slack_notification:
+    description: "Notify about failure on Slack"
+    steps:
+      - slack/status:
+          fail_only: true
+          # only_for_branches: master
```
Comment (richardkapolnai-da) [inline .circleci/config.yml:46]: oh, can't you enable this in order to ignore all PRs?

Comment (gaborh-da) [reply .circleci/config.yml:46]: that was the goal, but first tested it

Comment (gaborh-da) [reply .circleci/config.yml]: Done

File: .circleci/config.yml [hunk 2]
```diff
@@ -199,3 +212,18 @@ workflows:
             - daml_test
             - mvn_test
           <<: *only-release-tags
+  weekly_test:
```
Comment (richardkapolnai-da) [inline .circleci/config.yml:215]:
```suggestion
  scheduled_test:
```
or periodic_test

File: .circleci/config.yml (hunk 1 above)
Comment (richardkapolnai-da) [reply .circleci/config.yml]: awesome-O

File: .circleci/config.yml (hunk 2 above)
Comment (gaborh-da) [reply .circleci/config.yml]: done

**Pull request merged**